department-of-veterans-affairs/caseflow
# Craig/appeals 21533

Resolves [APPEALS-21533](https://vajira.max.gov/browse/APPEALS-21533)

### Description
- Add sortVisns() method to sort the VISN options by name

### Acceptance Criteria
- [x] List of VAMCs presented in the 'Assign to VISN' task action will be alphabetized.
- [x] Search functionality will remain in the drop-down.

### Testing Plan
1. Go to any Program Office queue as a Program Office member (VHAPOUSER in demo)
2. Go to any assigned case's details page 
3. In the actions dropdown, select "Assign to VISN"
4. Select "VA Medical Center" radio button
5. Verify that VISN options in the dropdown are sorted alphabetically and that the dropdown can be searched
6. Select a VAMC and verify that the VISN below the dropdown is correct for that VAMC
8. Select "VISN" radio button
9. Verify that the VISN list is in numerical order

### User Facing Changes

 BEFORE
 ![21533-before](https://user-images.githubusercontent.com/109101548/234377603-fd69dd67-aff1-4691-8873-9ff0de1c1ec7.png)

AFTER
![21533-after](https://user-images.githubusercontent.com/109101548/234377939-8ddfff4c-d4bb-461d-b5d7-ca477dbbc0ae.png)
